### DISCUSSION TITLE
Install RKE2 in /opt instead of /var/lib

### DISCUSSION THREAD

**Discussion by athulg93** [rancher/rke2#6426] [Q&A]
How do i deploy RKE2 such that it has nothing to do with /var/lib or any other the other system partitions..
I would like to have RKE2 running only on a specified partition that is /opt

Context:
we have 2 drives on our systems
SDA for the OS: 300 gb or so
SDB (/opt is mounted fully on SDB) for the product: Above 1 TB

I want to ensure that RKE2 and the product layer above it gets installed fully on SDB only

**Comment by brandond** [comment]
https://docs.rke2.io/reference/server_config#common
> data-dir | Folder to hold state | "/var/lib/rancher/rke2"
> -- | -- | --

**Reply by athulg93** [reply]
Hey,
With this we were able to move the content of the RKE2 to the folder in /opt.
But there are containers are still coming up in /var/lib
![Uploading Screenshot 2024-07-31 at 2.31.04 PM.png…]()

**Reply by brandond** [reply]
You screenshot is broken. Do you perhaps have some pods that are hardcoded to use that path?

**Reply by athulg93** [reply]
Hey, 
So we took the easy way out and decided to just mount the directory /var/lib onto SDB instead of trying all this.. it seems to work for us
